README.md

```diff
@@ -452,6 +453,14 @@ If set, the agent will use this proxy server to access the internet. These value
 *will* override the `http_proxy` or `https_proxy` environment variables. Lastly,
 `HttpProxy.Host` is required (if to be used) and `HttpProxy.Port` is optional.
 
+#### __CGroups.EnforceLimits__
+
+_Type: Boolean_  
+_Default: y_
+
+If set, the agent will attempt to set cgroups limits for cpu and memory for the agent process itself
+as well as extension processes. See the wiki for further details on this.
```

Comment:
`...See the wiki...` - I look forward to the verbiage.

Reply:
:+1: